ci-operator/jobs/openshift-kni/performance-addon-operators/openshift-kni-performance-addon-operators-release-4.11-postsubmits.yaml

```diff
@@ -13,7 +13,7 @@ postsubmits:
       ci.openshift.io/generator: prowgen
       job-release: "4.11"
     max_concurrency: 1
-    name: branch-ci-openshift-kni-performance-addon-operators-master-images
+    name: branch-ci-openshift-kni-performance-addon-operators-release-4.11-images
```

Comment:
master should be renamed 4.12 isn't it?

Reply:
this is the 4.11 postsubmits job. there is another job for master

Reply:
And the filename as well.

Reply:
This is an outcome of creating a 4.11 copy : https://github.com/openshift/release/pull/33526/files#diff-845d53be408e2b5ce81c7d5f2fa10af36677706373f9a2e68d54607a66c42350  from the master one.
Since we branched we should now have a 4.11 job added in addition to the master job